tla-types/src/main/scala/at/forsyte/apalache/tla/typecheck/etc/ToEtcExpr.scala

```diff
@@ -820,6 +820,39 @@ class ToEtcExpr(
         val opsig = OperT1(operArgs, a)
         mkExRefApp(opsig, Seq(v, variantEx))
 
+      case OperEx(VariantOper.`variantUnwrap`, tag @ _, _) =>
+        throw new TypingInputException(s"The first argument of VariantGetOnly must be a string, found: $tag", ex.ID)
+
+      case OperEx(VariantOper.variantGetUnsafe, v @ ValEx(TlaStr(tagName)), variantEx) =>
```

Comment:
how come just the first attribute is in backticks?

Reply:
For some reason, Intellij keeps doing that :)